select_test.go

```diff
@@ -165,10 +173,10 @@ func TestSelectBuilderParamJoin(t *testing.T) {
 
 func TestSelectBuilderNestedSelectJoin(t *testing.T) {
 
-	expectedSql := "SELECT * FROM bar JOIN ( SELECT * FROM baz WHERE foo = ? ) r ON bar.foo = r.foo"
-	expectedArgs := []interface{}{42}
+	expectedSql := "SELECT * FROM bar JOIN ( SELECT * FROM baz PREWHERE foo = ? WHERE foo = ? ) r ON bar.foo = r.foo"
```

Comment:
Add test with only `PREWHERE` and no `WHERE`

Reply:
Done